web/src/compose_validate.ts

```diff
@@ -409,6 +409,23 @@ export function warn_if_topic_resolved(topic_changed: boolean): void {
     // Pass topic_changed=true if this function was called in response
     // to a topic being edited.
 
+    if (
+        compose_state.get_is_processing_forward_message() &&
+        !compose_state.get_is_stream_recipient_dropdown_for_forward_message_closed()
+    ) {
+        // This is for the case of forwarding a message when the
+        // channel picker is opened. There is a possibility that
+        // this banner might be displayed at the same time the
+        // channel picker is opened. We don't want that situation.
+        // Therefore, we are preventing the display of this
+        // banner when the channel picker is opened.
+        return;
+    }
+    // We are resetting the state of the
+    // `is_stream_recipient_dropdown_for_forward_message_closed`,
+    // otherwise this will never become false.
+    compose_state.set_is_stream_recipient_dropdown_for_forward_message_closed(false);
```

Comment:
This is the wrong place to do this. I would name it with `open` and reverse the meaning, and just set it when opening that dropdown. As it is, this variable has a poorly-defined meaning that's likely to be quite fragile.